src/structures/Battlemetrics.js

```diff
@@ -667,6 +669,11 @@ class Battlemetrics {
             if (content['status'] === false) this.offlinePlayers.push(playerId);
         }
 
+        const rustWipes = details.rust_wipes;
```

Comment:
Place this closer to the other evaluation checks, below the rustmap evaluation check instead of down here